src/main/java/lotto/controller/LottoController.java

```diff
@@ -0,0 +1,84 @@
+package lotto.controller;
+
+import java.util.List;
+import lotto.constant.LottoRanking;
+import lotto.generator.RandomUniqueListGenerator;
+import lotto.model.Lotto;
+import lotto.model.LottoGameManager;
+import lotto.model.LottoOwner;
+import lotto.model.LottoResult;
+import lotto.model.PurchasePrice;
+import lotto.model.WinningLotto;
+import lotto.view.InputView;
+import lotto.view.OutputView;
+
+public class LottoController {
+    private final InputView inputView = new InputView();
+    private final OutputView outputView = new OutputView();
+
+    public void run() {
+        LottoOwner lottoOwner = getLottoOwner();
+        outputView.printLottosInfo(lottoOwner.getLottosInfo());
+        Lotto winningNumbers = getWinningNumbersUntilValid();
+        outputView.printSpace();
+        WinningLotto winningLotto = getWinningLottoUntilValidBonusNumber(winningNumbers);
+        LottoGameManager lottoGameManager = LottoGameManager.createGameManager(lottoOwner, winningLotto);
+        lottoGameManager.matchLottosWithWinningLotto();
+        printWinningStatistics(lottoGameManager);
+        outputView.printRateOfReturn(lottoGameManager.getRateOfReturn());
+    }
```

Comment:
메서드를 분리해 볼 수 있을 것 같아요! 😄

Reply:
공감합니다 힘들어보여요...

Reply:
개발 진행 중일때는 아래처럼 잠시 주석으로 분리하고, 기능 구현이 완료되셨을 떄 메서드 분리하시면 좋을 듯 합니다!
```suggestion
    public void run() {
        // 구입 금액 입력 및 로또 정보 출력
        LottoOwner lottoOwner = getLottoOwner();
        outputView.printLottosInfo(lottoOwner.getLottosInfo());
        
        // 당첨 번호 입력
        Lotto winningNumbers = getWinningNumbersUntilValid();
        outputView.printSpace();
        
        // 당첨 조회
        WinningLotto winningLotto = getWinningLottoUntilValidBonusNumber(winningNumbers);
        LottoGameManager lottoGameManager = LottoGameManager.createGameManager(lottoOwner, winningLotto);
        lottoGameManager.matchLottosWithWinningLotto();
        
        // 당첨 통계 출력
        printWinningStatistics(lottoGameManager);
        outputView.printRateOfReturn(lottoGameManager.getRateOfReturn());
    }
```

Reply:
@kimwanyoung @jinkshower @geoje  15라인 밑으로 작성되었다고 안일하게 넘어간 부분인거 같습니다!
놓친 부분이네요 피드백 감사합니다!